projboard/models.py

```diff
@@ -19,11 +19,68 @@ def get_subject_by_name(name):
         :return: Subject object
         """
         try:
-            subject = User.objects.get(name=name)
+            subject = Subject.objects.get(name=name)
         except ObjectDoesNotExist:
             return None
         return subject
 
+    def __str__(self):
+        return f"{self.name}"
+
+    @staticmethod
+    def get_list_subjects_names():
+        """
+        Method get list subject names
+        :return: the list of subject
+        """
+        subjects_list = list(Subject.objects.all().values_list('name', flat=True))
+        return subjects_list
+
+    @staticmethod
+    def create_new_subject(name):
+        """
+        Method create a new subject
+        :param name: The name of the subject
+        :return: the subject that created
+        """
+        try:
+            subject = Subject.objects.get(name=name)
+            return subject
+        except Subject.DoesNotExist:
+            subject = Subject(name=name)
+            subject.save()
+            return subject
+
+    @staticmethod
+    def edit_subject(existing_name, new_name):
```

Comment:
Consider renaming the method to `rename_subject` as it is not editing the subject, its just renaming it.

Reply:
Any updates to this?

Reply:
@yaelnoyman1 Please reply here

Reply:
I changed the name to "rename_suject"